base/applications/regedit/childwnd.c

```diff
@@ -181,11 +168,8 @@ static void draw_splitbar(HWND hWnd, int x)
     ReleaseDC(hWnd, hdc);
 }
 
-/*******************************************************************************
- * finish_splitbar [internal]
- *
- * make the splitbar invisible and resize the windows
- * (helper for ChildWndProc)
+/**
+ * make the splitbar invisible and resize the windows (helper for ChildWndProc)
```

Comment:
```suggestion
 * Make the splitbar invisible and resize the windows (helper for ChildWndProc)
```

Reply:
No, I don't want to change this for no reason / no gain. Remember that I do work on 8 branches in parallel. Not just on one. And this suggestion doesn't make anything better than today.